content/en/internal/getting-started/_index.md

```diff
@@ -1,14 +1,22 @@
 ---
 title: "Getting Started"
 linkTitle: "Getting Started"
+description: "Introduction to internal EGI services"
 type: docs
 weight: 10
 ---
 
-Please find the list of internal services on the
-[EGI website](https://www.egi.eu/internal-services/).
+Use this section to get started quickly with internal EGI services:
+
+1. The complete list of internal EGI services is
```

Comment:
why a numbered list?

Reply:
no particular reason. reverting to a bullet list.